### PR TITLE
Updated script line

### PR DESCRIPTION
**Pull request by joinimran**
There was typo in the script variable and has been fixed.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/4084

### CODE REVIEW

**Comment by jvsam** [comment]
@nenonix & @JohanFreelancer9 for copy review please. Thank you.

**Comment by jvsam** [comment]
Hi @DulceMontemayor this pull request has been copy edited and is ready for your final review and approval. Please let us know if there are outstanding issues to be resolved prior to merge. Thank you. See: [commits](https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4436/commits) and [changes](https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4436/files).

**Comment by jvsam** [comment]
Hi again @DulceMontemayor I'd like to follow-up on this PR. It contains suggestions that will fix the typo error in the script and remove the extra asterisk. Thank you so much. CC: @Dansimp 

P.S. I also asked @joinimran to fix the markdown syntax problem. Removing the extra html tag appeared to have fixed it on other issues.

image

File: windows/security/identity-protection/credential-guard/credential-guard-manage.md [hunk 1]
```diff
@@ -115,7 +115,7 @@ You can also enable Windows Defender Credential Guard by using the [Windows Defe
 DG_Readiness_Tool_v3.5.ps1 -Enable -AutoReboot
 ```
 > [!IMPORTANT]
-> When running the Windows Defender Device Guard and Windows Defender Credential Guard hardware readiness tool on a non-English operating system, within the script, change `*$OSArch = $(gwmi win32_operatingsystem).OSArchitecture` to be `$OSAch = $((gwmi win32_operatingsystem).OSArchitecture).tolower()` instead, in order for the tool to work. 
+> When running the Windows Defender Device Guard and Windows Defender Credential Guard hardware readiness tool on a non-English operating system, within the script, change `$OSArch = $(gwmi win32_operatingsystem).OSArchitecture` to be `$OSAch = $((gwmi win32_operatingsystem).OSArchitecture).tolower()` instead, in order for the tool to work.
```
Comment (illfated) [inline windows/security/identity-protection/credential-guard/credential-guard-manage.md]: I presume there is a typo in this line still, `$OSArch` is the environment variable ("$OSAch" without the R is incorrect):
```suggestion
> When running the Windows Defender Device Guard and Windows Defender Credential Guard hardware readiness tool on a non-English operating system, within the script, change `$OSArch = $(gwmi win32_operatingsystem).OSArchitecture` to be `$OSArch = $((gwmi win32_operatingsystem).OSArchitecture).tolower()` instead, in order for the tool to work. 
```

**Comment by illfated** [comment]
Also, as @jvsam already mentioned in https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4436#issuecomment-513494311 , please remove the lines 109 + 202 containing

```html
<span id="hardware-readiness-tool" />
```
```html
<span id="turn-off-with-hardware-readiness-tool" />
```
because these 2 HTML code lines both cause their respective following section heading to stop working as MarkDown code.

And if you can find time to do a cosmetic improvement after applying the remaining edits to this file, feel free to remove at least 4 of the 5 blank lines at the end of the file.

**Comment by illfated** [comment]
Oh. I see. You selected to add a blank line after `<span id="hardware-readiness-tool"/>` to make the heading line work again.
Fair enough. I don't quite know what that tag line does, so I will leave that for anyone else to work out.

I would like to suggest that you do the same to your line 203 too. That is the following line:
```
<span id="hardware-readiness-tool"/>
```

**Comment by jvsam** [comment]
@joinimran can you also correct the other headings please? Thanks for promptly working on this.

**Comment by joinimran** [comment]
Fixed for line 202. Ready for a review.

**Comment by martyav** [comment]
>I don't quite know what that tag line does, so I will leave that for anyone else to work out.

Writers use them so they can link to that section of the doc from other files. I think you can get the same functionality from Markdown headings. I joined in April, but I'm guessing the team used to do all/most of the formatting with HTML...

**Comment by illfated** [comment]
Good point. Myself, I started following this repository around November-December 2018, but I don't know how far back MS moved to Github and opened some repositories to ordinary users. I can only concur with your conclusion that most of the documentation process must have been migrated from a pure HTML-based platform.

**Comment by jvsam** [comment]
Thanks for pushing the additional updates@joinimran and for assistance @illfated. I believe this is good to go.

@martyav will you be able to help with the merge? Other than the cosmetic changes, the critical issue that is fixed in this PR is typo in the script. Thank you. 

CC: @DulceMontemayor

**Comment by jvsam** [comment]
Thanks for approving @Dansimp and @illfated. 

@Dansimp can you please also merge this PR? Thank you.

**Comment by jvsam** [comment]
Awesome! Thanks for merging @martyav !

**Pull request merged**